model-optimizer/extensions/ops/prelu.py

```diff
@@ -40,12 +39,6 @@ def __init__(self, graph: Graph, attrs: dict):
             'out_ports_count': 1,
         }, attrs)
 
-    def supported_attrs(self):
```

Comment:
We have this huge list of attrs somwhere during emitting. Means if the node has some attr set -- it will be emmited to IR. Lets have it as:
```
def supported_attrs(self):
    return []
```

Reply:
I am not sure what you are saying. The default implementation (in the `Op` class) is the following:
```
    def supported_attrs(self):
        """
        Attributes that user should/can set for the operation
        """
        return []
```

So in this case this default method will be called. Do I miss something?